Monika After Story/game/zz_music_selector.rpy

```diff
@@ -213,6 +213,31 @@ init -1 python in songs:
         if chan.mixer in renpy.game.preferences.volumes:
             renpy.game.preferences.volumes[chan.mixer] = _sanitizeVolume(value)
 
+    def addSongEntry(path, display_name, by_user=False):
+        """
+        Adds song entry to selection list. Automatically paginates and cleans
+        title text.
+
+        NOTE: Does not perform loop/metadata prefix scan. Path will be added as-is.
+        NOTE: Checks if the path exists (loop prefix is ignored for this check.)
+
+        IN:
+            path - path to the music file
+            display_name - title of the song to show in music selector
+            by_user (default False) - whether this is an action user has done
+                (sets the corresponding PM variable)
+        """
+
+        filepath = path.split(">")[:-1]
+        if not os.path.exists(filepath):
+            raise ValueError("{0} does not exist.".format(filepath))
```

Comment:
if we're expecting this to be called in topics, then this shouldnt raise an error. just have this return true/false depending on success/failure and let the caller (like a topic) decide how to gracefully handle the failure.

Reply:
Now returns `False` when there isn't such file at the given path.